src/Components/RemediationsPage/Components/LabelIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { Label } from '@patternfly/react-core/dist/esm/components/Label/index';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LabelIcon = ({ icon, text }) => (
```

Comment:
Docstring please, not sure we need separate component, probably could name it better to reflect the modifications (compact, outline and blue) .. `CompactOutlineBlueLabel` :sweat_smile:  ? the icon could be just passed as props because every label can have icon

Reply:
Also, those type of components that are kinda `common` along all other `pages` that we will do, can be moved to the `src/Components/Customs`.